### ISSUE TITLE
Unable to get slimpd installed on Ubuntu

### ISSUE THREAD

**Issue by brendan-pike** [othmar52/slimpd#52]
I've followed the instructions, when I get to the import;
var/www/slimpd $ sudo -uwww-data php -d memory_limit=-1 slimpd hard-reset
PHP Notice:  Undefined index: timezone in /var/www/slimpd/slimpd on line 58
PHP Notice:  date_default_timezone_set(): Timezone ID '' is invalid in /var/www/slimpd/slimpd on line 58
PHP Notice:  Undefined index: cli in /var/www/slimpd/slimpd on line 64
PHP Fatal error:  Allowed memory size of 2097152 bytes exhausted (tried to allocate 65536 bytes) in /var/www/slimpd/core/vendor-dist/illuminate/support/Collection.php on line 141

Please help.

**Comment by othmar52** [comment]
@brendan-pike 
thank you for reporting! i have a vague idea why this error occurs but i am not sure about. please pull the latest commit and give it another try.

**Comment by othmar52** [comment]
@brendan-pike giving feedback on issues you opened (thanks for that!!!) seems not to be your strength :)

Please can you tell me if the problem still exists?

**Comment by brendan-pike** [comment]
I'm so sorry, not sure what happened with my head.  I'll try this very soon and re-try (promise)

**Comment by brendan-pike** [comment]
The process appears to hang, eg.
slimpd # sudo -uwww-data php -d memory_limit=-1 slimpd hard-reset --force
All data of database:slimpd will be lost! Continue? [y/n]
y
Dropping database
Recreating database
Will migrate: Sun, 17 Jul 2016 09:53:46 +0000  (slimpd_v1) ---> Fri, 07 Apr 2017 16:14:34 +0000  

APPLY:    Fri, 06 Jan 2017 06:44:30 +0000  
NOW AT:   Fri, 06 Jan 2017 06:44:30 +0000  
Deleting files and directories inside localdata/cache/
Deleting files and directories inside localdata/embedded/
Deleting files and directories inside localdata/peakfiles/
STARTING import phase 0 triggerImport()
STARTING import phase 1 waitForMpd()
mpd seems to be ready. continuing...
FINISHED import phase 1 waitForMpd()
STARTING import phase 2 processMpdDatabasefile()

At this poin its using 100% CPU on a core but nothing happens even after an hour or so.

**Comment by othmar52** [comment]
please try to increase verbosity level in  `core/config/config_local.ini`
```ini
[config]
cli-verbosity = 10
```
due to some caching issues remove the cached config manually
```
rm localdata/cache/conf*
```
and give it a try

**Comment by brendan-pike** [comment]
I cleared the cache and up'd the verbosity but the same thing happened and with no additional CLI information.

**Comment by othmar52** [comment]
Maybe your webserver-user is not able to access your configured mpd-database file!?

Are you able to access sliMpd's systemcheck in your browser?

**Comment by brendan-pike** [comment]
Okay, I'm starting to get some perspective here, thank you.  My mpd is remote, is that a show stopper?

**Comment by othmar52** [comment]
Yes, unfortunately sliMpd will not work when those [system requirements](https://github.com/othmar52/slimpd/wiki/Installation#requirements) are not met

> Webserver (Apache, Nginx, Lighttpd, ...)
> **read-access to MPD-database-file (plain-text or gzip supported)**
> **read-access to filesystem configured in 'mpd.conf:music_directory'**

**Comment by brendan-pike** [comment]
I'll try and symlink those directories and see if it works.
